pangeo-notebook/conda-linux-64.lock

```diff
@@ -212,6 +216,8 @@ https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-htmlhelp-1.0.3-py_0.
 https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-jsmath-1.0.1-py_0.tar.bz2#67cd9d9c0382d37479b4d306c369a2d4
 https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-qthelp-1.0.3-py_0.tar.bz2#d01180388e6d1838c3e1ad029590aa7a
 https://conda.anaconda.org/conda-forge/noarch/sphinxcontrib-serializinghtml-1.1.4-py_0.tar.bz2#8ea6a8036e28dba8827d35c764709358
+https://conda.anaconda.org/conda-forge/linux-64/sqlalchemy-1.3.13-py38h516909a_0.tar.bz2#27c2d386486c21a4a134497abbe2d853
```

Comment:
Maybe not a problem, but mlflow pins to an older version of alembic, which requires this older SQLAlchemy.